include/kernels/unitarize_force.cuh

```diff
@@ -55,8 +55,8 @@ namespace quda {
         allow_svd(allow_svd),
         svd_only(svd_only),
         svd_rel_error(svd_rel_error),
-        svd_abs_error(svd_abs_error),
-        threads(u.VolumeCB(), 2, 1) { }
+        svd_abs_error(svd_abs_error), 
+	threads(u.VolumeCB(), 2, 1) { }
```

Comment:
formatting changed for the worse

Reply:
That is what I get for trying to meet the whitespace needs. It is possible to diff these with spacing ignored apparently. 
Since you will be clang-tidy-ing after anyway, can we do it that way
?

Reply:
`git diff` will show these formatting changes

Reply:
Hopefully fixed with b1cb6f61e